utils/schema/ordersSchema.js

```diff
@@ -0,0 +1,24 @@
+const mongoose = require('mongoose');
+const { config } = require('../../config');
+
+const { Schema } = mongoose;
+
+const ordersSchema = new Schema({
+  id_patient: Schema.Types.ObjectId,
+  id_doctor: Schema.Types.ObjectId,
+  is_complete: {
+    type: Boolean,
+    default: false,
+  },
+  created_at: Number,
+  updated_at: Number,
+}, {
+  timestamps: {
+    createdAt: 'created_at',
+    updatedAt: 'updated_at',
```

Comment:
@hyfi06 We have differences in this block compare with usersSchema, how do you want manage this dates?

Reply:
I can see in you new commit you fix dates. 😊

Reply:
I saw your implementation to have the same date management.